flow/scenarios/grid.py

```diff
@@ -512,21 +512,22 @@ def specify_intersection_edge_starts(self):
             [(":center", 0)]
         return intersection_edgestarts
 
-    def gen_even_start_pos(self, initial_config, num_vehicles, **kwargs):
+    @staticmethod
```

Comment:
@staticmethod is not needed for class methods, I think the correct @ is @classmethod?